MEMBERSHIP.md

```diff
@@ -5,8 +5,23 @@ to become a member. At the moment, the only privilege membership brings is being
 to vote for the project leader. And, of course, the glory of having your name appended
 to this file.
 
+## Applying
+
 To apply for membership, send a pull request to this file, adding your name and
-GitHub ID. 
+GitHub ID.
+
+Please mention why you became interested in SFOSC and share some related work that you
+have undertaken. This is optional though, please share only as much as you would like to.
+
```

Comment:
You can probably just say "and share some related work." No need to bring in... the [undertaker!](https://en.wikipedia.org/wiki/The_Undertaker)